apps/antalmanac/src/components/RightPane/CoursePane/CourseRenderPane.tsx

```diff
@@ -266,9 +266,9 @@ export default function CourseRenderPane(props: { id?: number }) {
      */
     useEffect(() => {
         return () => {
-            setHoveredCourseEvents(undefined);
+            setHoveredEvents(undefined);
         };
-    }, [setHoveredCourseEvents]);
+    }, [setHoveredEvents]);
```

Comment:
This shouldn't be in the dependency array since it's a setter, but ESLint might complain about that, so I guess it's alright to leave it if that's the case unfortunately.

```suggestion
    }, []);
```